lib/pages/search_page.dart

```diff
@@ -41,19 +41,10 @@ class SearchPage extends HookConsumerWidget {
                 Switch(
                   value: ref.watch(showPastEventsProvider),
                   onChanged: (value) {
-                    if (value) {
-                      ref
-                          .read(currentSortOrderProvider.notifier)
-                          .set(SortOrder.descending);
-                    } else {
-                      ref
-                          .read(currentSortOrderProvider.notifier)
-                          .set(SortOrder.ascending);
-                    }
```

Comment:
Nie rozumiem kto to dodał i po co ale wywalam bo psuje coś xD

Reply:
ja bym strzelal ze to Ty dodawales XD

Reply:
Już tłumaczę feature dodany celowo. W momencie gdy ogladasz tylko nadchodzace wydarzenia chcesz widziec  najpierw najbliższe a więc rosnące sortowanie, gdy dodasz stare, powinno zmienić się na malejąco. U mnie działało i nie zaobserwowałem problemów. Proszę o nie wywalanie

Reply:
Coś ewidentnie się psuje przez to bo jak jest odkomentowane to po użyciu toggla przez chwilę jest asyncError xD. nie jest to jakiś duży problem bo potem się ładuje normalnie ale coś jest nie halo. Jak szukałem to chyba to jest bug w riverpodzie o którym już wiedzą a nie problem ze nasze jest chujowe. Mogę spróbować jakoś obejść tego buga ale trochę nie widzę sensu w funkcjonalności, nie zważając że taki behavior powinien się dziać tylko gdy sortujemy po dacie bo dla nazwy to nie ma sensu.